checks.d/yarn.py

```diff
@@ -196,9 +197,19 @@ def _yarn_app_metrics(self, rm_address, addl_tags):
 
                     for app_json in metrics_json['apps']['app']:
 
-                        app_name = app_json['name']
+                        tags = []
+
+                        for tag in app_tags:
+                            try:
+                                key, value = tag.split(':')
+                                tags.append("{tag}:{value}".format(
+                                    tag=value, value=str(app_json[key])
+                                ))
+                            except ValueError:
+                                self.log.error("Invalid value {0} for application_tag").format(tag)
```

Comment:
The parenthesis is still not correct 😉 .
Also could you use the logger interpolation instead of doing it with `format` ? https://www.simonmweber.com/2014/11/24/python-logging-traps.html
It should be `self.log.error("Invalid value %s for application_tag", tag)`.